charts/values.yaml

```diff
@@ -0,0 +1,250 @@
+## Default values for charts.
+## This is a YAML-formatted file.
+## Declare variables to be passed into your templates.
+
+## @param global clusterpedia global config
+global:
+  ## @param dbInstallMode "internal" and "external" are provided
+  ## "internal" means automatic Database installation, default mysql
+  ## "external" means to install the database yourself, must specify the database connection parameters
+  storageInstallMode: "internal"
+  ## persistence configuration
+  ##
+  persistence:
+    ## @param global.persistence.matchNode define the nodeAffinity of pv.
+    ## if defined, will create the pv on the node
+    matchNode: kind-control-plane
```

Comment:
`storageInstallMode` and `persistence` don't seem to make sense for `global`, only for clusterepdia, there is no need to put them in `global`

Reply:
`imageRegistry` and `imagePullSecrets` look like they could be placed in global.
example:
```yaml
global:
## @param global.imageRegistry Global Docker image registry
##
imageRegistry: ""
## @param global.imagePullSecrets Global Docker registry secret names as an array
## e.g.
## imagePullSecrets:
##   - myRegistryKeySecretName
##
imagePullSecrets: []
```